src/client/components/Auth/AuthenticatedRoute.js

```diff
@@ -25,10 +25,10 @@ function AuthenticatedRoute({ children, requiredRole, ...rest }) {
           }
           // I am just adding here but i am not sure whether it is the right place
           if (userRole === requiredRole) {
-            return 'matches'; // no idea on what to do if it matches
+            return children; // no idea on what to do if it matches
           }
```

Comment:
Remove comments.
Have you tested what happens if you remove the code in `App.js` and still try to open this page?

Reply:
it shows 404.

Reply:
So it is obviously wrong. What should happen if there is no `requiredRole`?

Reply:
oh ya , it should not display 404, i changed